### ISSUE TITLE
Remove "Run..." shortcut from 7-style menu by default

### ISSUE THREAD

**Issue by bonzibudd** [Open-Shell/Open-Shell-Menu#559]
**Is your feature request related to a problem? Please describe.**
Beginning with Windows Vista, the Run dialog has been de-prioritized in favor of the search box, which can perform the same functions. However, the Windows 7 style menu in Open-Shell retains the shortcut from the older styles, which is not needed for most people. It can also take up an excessive amount of vertical space, and this is worsened on Windows 10 and 8, which require an additional shortcut to Settings:
![image](https://user-images.githubusercontent.com/61938331/101284556-87f8d180-37ae-11eb-8208-c5d862062dee.png)

**Describe the solution you'd like**
The "Run..." shortcut should be removed from the right-hand side by default, but should obviously remain available to enable under "Customize Start Menu". The Classic and Two columns preferences should remain unchanged.

**Additional context**
This might be more preferential, but I believe it will be better for new users to have the option disabled by default. If it is needed, users can always go to the "Customize Start Menu" tab and enable it there. This also shouldn't have any affect on existing installations or XML files.

**Comment by ge0rdi** [comment]
Personally I'd not remove the item by default (one can easily remove it).

While search box provides more natural and in many cases better way of running programs, it is for example not possible to run program with arguments from search box.

On the other hand, `Win+R` shortcut provides even easier access to Run dialog than Start menu item.

I guess we should just wait for more feedback on this.

**Comment by bonzibudd** [comment]
That is true, they each have their own uses, but I agree that it's probably easier in general to use Win+R. I personally think it would be nice to more closely match Windows 7 by default, as it is made to emulate it. Then again, it might not be that big of a deal in the first place.
